modules/power-regulators.stanza

```diff
@@ -57,8 +59,10 @@ for v in tlv do:
       schematic-group(ps) = power
       ;Create nets
       net (ps.gnd, neg(input), neg(output))
-      net (ps.vin, pos(input), ps.en)
-      net (ps.vout, pos(output))
+      net vin (ps.vin, pos(input), ps.en)
+      net vreg (ps.vout, pos(output))
+      property(vreg.net-voltage) = ocdb/tolerance/typ(voltage(output))
+      property(vin.net-voltage) = ocdb/tolerance/typ(voltage(input))
```

Comment:
Why would you not assign 0.0v to the ground here?
net gnd (ps.gnd neg(input), neg(output))

property(gnd.net-voltage) = ocdb/tolerance/typ(0.0)

Reply:
That could be assigned here, yes